File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int>arr(26,0);
        for(int i=0;i<s.length();i++){
            arr[s[i]-'a']++;
        }
        for(int i=0;i<t.length();i++){
            arr[t[i]-'a']--;
        }
        for(int i=0;i<arr.size();i++){
            if(arr[i]!=0)
            return false;
        }
        return true;
        // sort(s.begin(),s.end());
        // sort(t.begin(),t.end());
        // return s==t?true:false;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution(object):
    def isAnagram(self, s, t):
        h=[0]*26
        for i in range(len(s)):
            h[ord(s[i])-ord('a')]+=1
        for i in range(len(t)):
            h[ord(t[i])-ord('a')]-=1
        for i in h:
            if i!=0:
                return False
        return True
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        